Azure/sonic-buildimage
# Fix for TSA error logging on multi-asic

#### Why I did it
TSA/TSB scripts check current state and report error if the state is already in Maintenance/Normal mode. This lookup was not done per DB instance

#### How I did it
Changed state lookup per DB instance

#### How to verify it
Execute TSA twice on multi-asic and check that the error message - "System already in Maintenance" is logged for each DB instance

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Link to config_db schema for YANG module changes


TSA/B script output on Multi-asic
[log_and_err_message_output.txt](https://github.com/Azure/sonic-buildimage/files/9226405/log_and_err_message_output.txt)
